src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/.eslintrc.json

```diff
@@ -0,0 +1,176 @@
+{
+    "$schema": "http://json.schemastore.org/eslintrc",
+    "root": true,
+  "env": {
+    "es6": true,
+    "node": true
+  },
+  "extends": [
+    "plugin:@typescript-eslint/recommended",
```

Comment:
Is there no way to extend a single `.eslintrc.json`s? It seems quite troublesome to maintain N number of these.